include/igl/copyleft/comiso/nrosy.cpp

```diff
@@ -803,8 +805,16 @@ Eigen::VectorXd igl::copyleft::comiso::NRosyField::angleDefect()
     {
       Eigen::VectorXd a = V.row(F(i,(j+1)%3)) - V.row(F(i,j));
       Eigen::VectorXd b = V.row(F(i,(j+2)%3)) - V.row(F(i,j));
-      double t = a.transpose()*b;
-      t /= (a.norm() * b.norm());
+      double t = a.transpose() * b;
+      double norm_prod = a.norm() * b.norm();
+      if ( norm_prod > 0. )
```

Comment:
Hi. Can you remove the space inside the parentheses? `if (norm_prod > 0.)` (same for the rest of the PR, and for the throw)

Reply:
Right. I have to check the coding style later on. Sorry.